src/__tests__/integration/mod-translation-flow.test.ts

```
@@ -0,0 +1,278 @@
+                jarPath: args.jarPath,
+                langFiles: [
+                  { language: 'en_us', path: 'assets/custommod/lang/en_us.json', content: {} },
+                ],
+                patchouliBooks: [],
+                langFormat: 'json',
+              });
+            }
+          
+          case 'check_mod_translation_exists':
+            // Use test data to determine if translation exists
+            const testData = generateTestModData();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)